### ISSUE TITLE
"allow_editing" is false by default?

### ISSUE THREAD

**Issue by bennibu** [gollum/gollum#911]
When I upgraded to v3.1.0 I had to set the wiki_option `allow_editing` to true to re-enable editing. We use gollum as a rack application with the following config.ru:

``` ruby
....
Precious::App.set(:wiki_options, {
    universal_toc: false,
    allow_editing: true,
    live_preview: false,
    allow_uploads: false
  })

run Precious::App
```

If so, please update the README.md or HISTORY.md.

**Comment by dometto** [comment]
It turns out that this was not resolved completely by 0914959  (read all about it [here](https://github.com/gollum/gollum/pull/950)). I just merged and released the proper fix in `v3.1.2`.

**Comment by arr2036** [comment]
Thanks for keeping this issue updated :)
